cfme/utils/appliance/__init__.py

```diff
@@ -2519,7 +2527,7 @@ def configure_fleecing(self, log_callback=None):
                 self.add_rhev_direct_lun_disk()
 
             log_callback('Enabling smart proxy role...')
-            roles = self.server.settings.server_roles_db
+            roles = self.server.settingghos.server_roles_db
```

Comment:
a typo?

Reply:
Literally saw this earlier and thought I had fixed, thanks @rlbabyuk